src/XMakeTasks/ManifestUtil/SecurityUtil.cs

```diff
@@ -762,20 +762,33 @@ internal static string GetPathToTool(System.Resources.ResourceManager resources)
         {
 #pragma warning disable 618 // Disabling warning on using internal ToolLocationHelper API. At some point we should migrate this.
```

Comment:
I don't know the story on either. Can you file a bug for us to figure it out? Since you're not making things worse I think this is fine.